documentation/source/users/rmg/input.rst

```diff
@@ -549,6 +552,12 @@ Miscellaneous options::
         saveEdgeSpecies=True,
     )
 
+The ``name`` field is the name of any generated seed mechanisms
+
+Setting ``generateSeedEachIteration`` to ``True`` tells RMG to save and update a seed mechanism and thermo library during the current run
```

Comment:
Supposing it is 'off' It will make one seed at the beginning to initialize the folder and I believe that if the job fails in certain ways it will generate one at the end, but otherwise it will not update it.